joi-validations/topics.validations.js

```diff
@@ -5,6 +5,16 @@ const TopicValidation = {
     query: {
       name: string.required()
     }
+  },
+  checkName: {
+    body: {
+      name: string.required()
+    }
+  },
+  create: {
+    body: {
+      name: string.required()
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `checkName` and `create` validation objects are identical, both requiring a `name` in the body of the request. This is not necessarily an issue if both endpoints require the same validation. However, it's worth considering if the `create` endpoint should have additional validation requirements, such as other fields that might be necessary to create a new topic.

If there are additional fields required for creating a topic, you should add them to the `create` validation object. For example, if a `description` field is needed, the validation could look like this:

```diff
  create: {
    body: {
      name: string.required(),
+     description: string.required()
    }
  }
```

If no additional fields are required, then the code is fine as it is.


<!-- This is an auto-generated comment by OSS CodeRabbit -->